content/en/docs/components/model-registry/installation.md

```diff
@@ -0,0 +1,66 @@
++++
+title = "Installation"
+description = "How to set up Model Registry"
+weight = 20
+
++++
+
+This section details how to set up and configure Model Registry on your Kubernetes cluster with Kubeflow.
+
+## Prerequisites
+
+These are the minimal requirements to install Model Registry:
+
+- Kubernetes >= 1.27
+- Kustomize >= 5.0.3 ([see more](https://github.com/kubeflow/manifests/issues/2388))
+
+<a id="model-registry-install"></a>
+
+## Installing Model Registry
+
+You can skip this step if you have already installed Kubeflow >=1.9. Your Kubeflow
+deployment includes Model Registry ([see tracker issue](https://github.com/kubeflow/manifests/issues/2631)).
+
+To install Model Registry as part of Kubeflow, follow the
+[Kubeflow installation guide](/docs/started/installing-kubeflow/).
+
+If you want to install Model Registry separately from Kubeflow, or to get a later version
+of Model Registry, you can use one of the following Model Registry manifests.
+Remember to substitute the relevant release (e.g. `v0.1.2`), modify `ref=main` to `ref=v0.1.2`.
+
+The following steps show how to install Model Registry in the context of a default Kubeflow >=1.8 installation.
+
+```shell
+kubectl apply -k "https://github.com/kubeflow/model-registry/manifests/kustomize/overlays/db?ref=main"
```

Comment:
@tarilabs I think we should explain what this will do, and give "steps" in a clear way. E.g. what is the difference between this command and the next one?

We should also not target the `main` branch by default.

We can target a specific version tag, and link to the "releases" or "changelog" and say you can update it with a newer version.

Reply:
noted, in my immediate todo; thanks

Reply:
tracked in: https://github.com/kubeflow/model-registry/issues/109
proposed impl with: https://github.com/kubeflow/website/pull/3740

marking as resolved this thread accordingly.